### PR TITLE
Boilerplate Spec for master Branch

### PR DESCRIPTION
**Pull request by dblevins**
Generated boilerplate spec.  Feel free to merge, then refine.

 See https://wiki.eclipse.org/How_to_Prepare_API_Projects_to_Jakarta_EE_8_Release

### CODE REVIEW

File: spec/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+============================
+
+This project generates the Jakarta WebSocket Specification.
+
+Building
+--------
+
+Prerequisites:
+
+* JDK8+
```
Comment (joakime) [inline spec/README.md]: If this is to remain Java EE 8 compatible, then this JDK8 requirement is odd.
What is this JDK8 requirement for?  the API or the Spec?
(API has traditionally had a JDK6+ requirement for releases 1.0 and 1.1)

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        <spec_version>1.1</spec_version>
+        <!--<spec_build>02</spec_build>-->
+        <new_spec_version>1.1</new_spec_version>
+        <spec_impl_version>1.1.1</spec_impl_version>
+        <packages.export>javax.websocket.*</packages.export>
+    </properties>
+
+    <modules>
+        <module>api</module>
+        <module>spec</module>
```
Comment (joakime) [inline pom.xml]: This is compatible with the `api` module.
`api` requires JDK6+
We've had issues using JDK8 to target JDK6 bytecode.

File: spec/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* Maven 3.0.3+
```
Comment (joakime) [inline spec/README.md]: Maven 3.0.x is a strange choice.
Eg: if you use Java 9+ then you'll have issues.

File: spec/pom.xml [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>websocket-spec</artifactId>
+    <packaging>pom</packaging>
+    <name>Jakarta WebSocket Specification</name>
+
+    <properties>
+        <site.output.dir>${project.build.directory}/staging</site.output.dir>
+        <maven.site.skip>true</maven.site.skip>
+        <asciidoctor.maven.plugin.version>1.5.7.1</asciidoctor.maven.plugin.version>
+        <asciidoctorj.version>1.6.2</asciidoctorj.version>
```
Comment (joakime) [inline spec/pom.xml]: This is an ancient version of asciidoctor.
The Eclipse Jetty project (which also uses asciidoctor for documentation) discovered that it has issues on non-oracle JVMs (eg: ibm, and adoptopenjdk, and azul), newer versions of OSX (10.15+), and newer versions of Maven (3.5.0+).

**Comment by dblevins** [comment]
Hi @joakime !  I don't have commit on this project, so you or someone else would need to merge.  Note, the boilerplate was generated based on work from a couple active projects in efforts to help the other projects catchup.  We all need to complete the release process this month.

Consider the PR "hopefully helpful" at best and feel free to change absolutely anything.  If there are improvements that might apply to other projects, feel encouraged to socialize them on jakartaee-spec-project-leads@eclipse.org

**Pull request merged**